src/quicknxs/interfaces/event_handlers/main_handler.py

```diff
@@ -187,7 +188,9 @@ def file_loaded(self):
         self.main_window.file_loaded_signal.emit()
         self.main_window.initiate_reflectivity_plot.emit(False)
         self.main_window.initiate_projection_plot.emit(False)
-
+        self.status_bar_handler.show_message(
```

Comment:
Now it looks like the message will be shown after loading any reduced data file, whether the scale error column was missing or not. Could we return a flag from `read_reduced_file` and only show the warning if that column is missing? If the warning is always shown it is likely to be ignored...